### ISSUE TITLE
Wrong hierarchical level (WCAG 1.3.1a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1220]
The nesting of the hierarchy levels does not correspond to the content structure on both sides checked. There is twice a heading that refers to personal settings ("Persönliche Einstellungen" and "Profile Settings").

A heading h1 should be created that precedes the following heading levels (suggestion: "Eigenes Profil"). The headings "Persönliche Einstellungen", "Gruppen", "Analytics", "Bild austauschen" and "Ändere meine persönlichen Daten" should be assigned to level h2.

https://slidewiki.org/user/vivien.zschammer/settings/profile
https://slidewiki.org/user/vivien.zschammer/stats

**Comment by VivienDZB** [comment]
Sites: Editor + Markdown editor
https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/edit 
https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/markdownEdit

The h2 heading “Information about SlideWiki” could be assigned to the order h1 for a meaningful hierarchy. The associated sub-items should be marked with the element h2.
